Cubical/Foundations/Pointed/Properties.agda

```diff
@@ -1,44 +1,60 @@
 {-# OPTIONS --cubical --no-import-sorts --safe #-}
```

Comment:
This file would be a bit nicer with the levels declared at the top as variables. Can you fix?